### PR TITLE
Merging WIP to v2 Preview

### PR DESCRIPTION
**Pull request by JamesBauman**
Merging work-in-progress to the zowe preview docs website

@awharn : Can you please approve this PR.....  there is nothing to review...just merging my current work into the v2 preview release.

thx,
jb

### CODE REVIEW

File: sidebars.js [hunk 1]
```diff
@@ -220,14 +220,16 @@ module.exports = {
...
+              label: "Configuring team profiles",
+
               items: [
                 "user-guide/cli-using-configuring-global-profiles",
                 "user-guide/cli-using-initializing-global-configuration",
                 "user-guide/cli-using-creating-global-user-profiles",
                 "user-guide/cli-using-creating-global-team-profiles",
                               ],
-            },          
+
```
[the remark this answers is not in the document]
Comment (JamesBauman) [reply sidebars.js]: i restored the missing bracket...

File: sidebars.js [hunk 2]
```diff
@@ -220,15 +220,14 @@ module.exports = {
...
+              label: "Configuring team profiles",
               items: [
                 "user-guide/cli-using-configuring-global-profiles",
                 "user-guide/cli-using-initializing-global-configuration",
                 "user-guide/cli-using-creating-global-user-profiles",
                 "user-guide/cli-using-creating-global-team-profiles",
                               ],
-            },          
-            "user-guide/cli-using-integrating-apiml",
+            }            "user-guide/cli-using-integrating-apiml",
```
Comment (awharn) [inline sidebars.js:230]: This line is still missing a comma. 😅

Comment (JamesBauman) [reply sidebars.js]: my fingers are crossed....

**Review by awharn — APPROVED** [review]

**Comment by JamesBauman** [comment]
Signed-off-by: JamesBauman <james.bauman@broadcom.com>

**Pull request merged**